include/lsst/meas/base/PixelFlags.h

```diff
@@ -94,10 +110,12 @@ class PixelFlagsAlgorithm : public SimpleAlgorithm {
     ) const;
 
 private:
-
+    template <typename MaskedImageT>
+    class FootprintBits;
+    typedef afw::image::MaskedImage<float> MaskedImageT;
+    void updateFlags(const KeyMap *,const FootprintBits<MaskedImageT> *,afw::table::SourceRecord & ) const;
```

Comment:
`const` should come after the type name (coding standard 5-10).

Why is this taking pointers rather than references?

And does it need to be a member of `PixelFlagsAlgorithm`? Can we make it a free function in an anonymous namespace instead?